chess-AI/ai_io.py

```diff
@@ -7,7 +7,7 @@
 
 import torch
 
-from options import TrainOptions
+from options import *
```

Comment:
I realize in looking at [the style guide](https://google.github.io/styleguide/pyguide.html#22-imports) that the way we've been handling imports everywhere is not as we should, so I'll make an issue to fix it everywhere, but for now, can you change this to `import options` and then do `options.Blah` everywhere?